x/ccv/consumer/keeper/validators.go

```diff
@@ -13,42 +13,42 @@ import (
 )
 
 // ApplyCCValidatorChanges applies the given changes to the cross-chain validators states
```

Comment:
It should be commented that changes are filtered from an edge case. See my comment in
https://github.com/cosmos/interchain-security/issues/230#issuecomment-1186929234

Reply:
Good point, I'll add this.